cmd/internal/cli/verify.go

```diff
@@ -210,6 +222,13 @@ func doVerifyCmd(cmd *cobra.Command, cpath string) {
 		if err != nil {
 			sylog.Fatalf("Failed to load certificate: %v", err)
 		}
+
+		// reject self-signed certificates.
```

Comment:
I'm not sure about this being globally applied here. There are very valid reasons to use a self-signed certificate (outside of OCSP)... and if we block their use here I'd expect we will get issues raised by people testing things.